docs/bin/plugin_formatter.py

```diff
@@ -132,6 +133,17 @@ def rst_xline(width, char="="):
     return char * width
 
 
+def to_boolean(value):
+    """return default value in boolean type."""
```

Comment:
Since it's a sentence, start it with an uppercase letter. Also, it looks like the proper docstring would sound like `Normalize boolean option value.`

Reply:
ok